File: tool/src/ru/fomenkov/util/Log.java
package ru.fomenkov.util;

public class Log {

    private Log() {
    }

    public static void d(String format, Object... args) {
        String message = String.format(format, args);
        System.out.println(message);
    }

    public static void e(String format, Object... args) {
        String message = String.format(format, args);
        System.err.println(message);
    }
}

File: plugin/src/ru/fomenkov/runner/logger/Log.kt
package ru.fomenkov.runner.logger

object Log {

    fun d(message: String, newLine: Boolean = true) {
        if (newLine) {
            println(message)
        } else {
            print(message)
        }
    }

    fun e(message: String) = System.err.println(message)
}